src/Microsoft.PowerShell.Commands.Utility/commands/utility/EnableDisableRunspaceDebugCommand.cs

```diff
@@ -351,7 +351,7 @@ public SwitchParameter BreakAll
         }
```

Comment:
@KirkMunro, your last commit had 1 failures in `PowerShell-CI-windows`
Tests for (error, warning, etc) action preference.action preference of Ignore can be set as a preference variable using an enumerated value
```powershell
The value Ignore is not supported for an ActionPreference variable. The provided value should be used only as a value for a preference parameter, and has been replaced by the default value. For more information, see the Help topic, "about_Preference_Variables."
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\ActionPreference.Tests.ps1: line 53
53:                 Get-Process -Name asdfasdfasdf

```